src/install-edge-runtime.sh

```diff
@@ -31,8 +31,8 @@ function install_edge_runtime() {
 
     if [ -x "$(command -v iotedge)" ];
     then
-        log_error "Edge runtime is already available."
-        exit ${EXIT_CODES[9]}
+        log_info "Edge runtime is already available, removing it"
+        sudo apt purge aziot-edge  -y 2>>$STDERR_REDIRECT 1>>$STDOUT_REDIRECT &
```

Comment:
>sudo apt purge aziot-edge [](http://example.com/codeflow?start=8&length=25)

the script should not be uninstalling iot edge. if the runtime has already been installed, the script should exit, this is one of the requirements. Instead we should do a cleanup step in our pipeline, the steps are at this wiki - [https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fteams.microsoft.com%2Fl%2Fchannel%2F19%253AsZ1aFw8VCGOThBLr3Q591UNcsl56rsy06urzp3ulHrw1%2540thread.tacv2%2Ftab%253A%253A83f98411-21c7-4d30-8304-a6e2faafd8e6%3FgroupId%3D268bf7c3-fe59-456b-b6d5-e79e6eb0aaca%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=04%7C01%7CMariana.Mihova%40microsoft.com%7C44211c6efadf44f02f7008d9262f5e60%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637582807262088002%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=hpym1UvYZxse0AmEMlmQY1urOo3QBUuT8zl2Rfl1VT8%3D&reserved=0](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fteams.microsoft.com%2Fl%2Fchannel%2F19%253AsZ1aFw8VCGOThBLr3Q591UNcsl56rsy06urzp3ulHrw1%2540thread.tacv2%2Ftab%253A%253A83f98411-21c7-4d30-8304-a6e2faafd8e6%3FgroupId%3D268bf7c3-fe59-456b-b6d5-e79e6eb0aaca%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=04%7C01%7CMariana.Mihova%40microsoft.com%7C44211c6efadf44f02f7008d9262f5e60%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637582807262088002%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=hpym1UvYZxse0AmEMlmQY1urOo3QBUuT8zl2Rfl1VT8%3D&reserved=0)